src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,169 @@
+
+  const handleOpenMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(true);
+  };
+
+  const handleNotificationsMenu = (event) => {
+    setAnchorElNotifications(event.currentTarget);
+  };
+
+  const handleNotificationsClose = () => {
+    setAnchorElNotifications(null);
+  };
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static" color="primary">
+        <Toolbar>
+          {session.info.isLoggedIn ? (
+            <>
+              <Typography variant="h4" noWrap component="div" mr="10px">
+                PASS
+              </Typography>
+              <>
+                <Link to="/PASS/home/">
+                  <Button
+                    variant="contained"
+                    aria-label="outlined primary button group"
+                    color="tertiary"
+                  >
+                    Home
+                  </Button>
+                </Link>
+                <Link to="/PASS/forms/">
+                  <Button
+                    variant="contained"
+                    aria-label="outlined primary button group"
+                    color="tertiary"
+                  >
+                    Forms
+                  </Button>
+                </Link>
+                <Box sx={{ flexGrow: 1 }} />
+                <IconButton
+                  size="large"
+                  aria-label="show 17 new notifications"
+                  color="inherit"
+                  onClick={handleNotificationsMenu}
+                >
+                  <Badge badgeContent={17} color="error">
+                    <NotificationsIcon />
+                  </Badge>
+                </IconButton>
+                <Menu
+                  id="menu-appbar-notifications"
+                  anchorEl={anchorElNotifications}
+                  anchorOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right'
+                  }}
+                  keepMounted
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right'
+                  }}
+                  open={Boolean(anchorElNotifications)}
+                  onClose={handleNotificationsClose}
+                >
+                  <MenuItem onClick={handleNotificationsClose}>
+                    <p>Notification 1</p>
+                  </MenuItem>
+                  <MenuItem onClick={handleNotificationsClose}>
+                    <p>Notification 2</p>
+                  </MenuItem>
+                </Menu>
+                <IconButton
+                  size="large"
+                  edge="end"
+                  aria-label="account of current user"
+                  aria-controls={menuId}
+                  aria-haspopup="true"
+                  onClick={handleOpenMenu}
+                  color="inherit"
+                >
+                  <AccountCircle />
+                </IconButton>
+                <Menu
```

Comment:
Line 121 to Line 142 looks duplicated from Line 88 to 109. Keep one of them, but not both.